invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/SearchApp.js

```diff
@@ -55,9 +61,15 @@ export const SearchAppResultsPane = ({ layoutOptions }) => {
 
 export const SearchApp = ({ config, appName }) => {
   const searchApi = new InvenioSearchApi(config.searchApi);
+  const _buildUID = (element) => buildUID(element, "", appName);
```

Comment:
I am not sure I understand why this is needed: isn't it the global app namespace enough?

Reply:
Sorry, I didn't understand the question :/ The `appName` is the one passed to the component thus the global app namespace. I just defined this small function because I need to duplicate the call `buildUID(element, '', appName)` in the SearchApp component...